server/src/main/java/com/showmetheroapes/paracord/controllers/StrandController.java

```diff
@@ -0,0 +1,24 @@
+package com.showmetheroapes.paracord.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/v1/strands", produces = "application/json")
+public class StrandController {
+
+  /**
+   * Endpoint to get all of the currently available strands
+   *
+   * @return a list of strand names
+   */
+  @GetMapping
+  public List<String> findAll() {
+    final List<String> strands = new ArrayList<>();
+    strands.add("Test");
+    return strands;
+  }
```

Comment:
We talked about this already, but just to verify that I'm remembering correctly - If we wanted to make a POST controller for the same URL, we would make another public function here and precede it with `@PostMapping`, right?

Reply:
Yes